drivers/led_strip/ws2812_spi.c

```diff
@@ -182,8 +194,18 @@ static const struct led_strip_driver_api ws2812_spi_api = {
 	(DT_INST_PROP(idx, spi_one_frame))
 #define WS2812_SPI_ZERO_FRAME(idx) \
 	(DT_INST_PROP(idx, spi_zero_frame))
+/* Get the latch/reset delay from the "reset-delay" DT property. */
+#define WS2812_RESET_DELAY(idx) \
```

Comment:
The new line above is not needed. Please keep this line as is.

Reply:
Agree, adjusted